cmd/sebak/cmd/run.go

```diff
@@ -194,6 +210,39 @@ func parseFlagsNode() {
 		common.PrintFlagsError(nodeCmd, "--storage", err)
 	}
 
+	var tmpInt int
+	if tmpInt, err = strconv.Atoi(flagTimeoutINIT); err != nil {
+		common.PrintFlagsError(nodeCmd, "--timeout-init", err)
+		timeoutINIT = 0
+	} else {
+		timeoutINIT = time.Duration(tmpInt) * time.Second
+	}
+
+	if tmpInt, err = strconv.Atoi(flagTimeoutSIGN); err != nil {
+		common.PrintFlagsError(nodeCmd, "--timeout-sign", err)
+		timeoutSIGN = 0
+	} else {
+		timeoutSIGN = time.Duration(tmpInt) * time.Second
+	}
+
+	if tmpInt, err = strconv.Atoi(flagTimeoutACCEPT); err != nil {
+		common.PrintFlagsError(nodeCmd, "--timeout-accept", err)
+		timeoutACCEPT = 0
+	} else {
+		timeoutACCEPT = time.Duration(tmpInt) * time.Second
+	}
+
+	if tmpInt, err = strconv.Atoi(flagTimeoutALLCONFIRM); err != nil {
+		common.PrintFlagsError(nodeCmd, "--timeout-allconfirm", err)
+		timeoutALLCONFIRM = 0
+	} else {
+		timeoutALLCONFIRM = time.Duration(tmpInt) * time.Second
+	}
+
+	if transactionsLimit, err = strconv.Atoi(flagTransactionsLimit); err != nil {
+		common.PrintFlagsError(nodeCmd, "--transactions-limit", err)
+	}
+
 	if threshold, err = strconv.Atoi(flagThreshold); err != nil {
 		common.PrintFlagsError(nodeCmd, "--threshold", err)
 	}
```

Comment:
@Charleslee522 We need to check the timeout and limit values is over 0. It can be parsed as under 0.

Reply:
@Charleslee522 👍